swagger/scylla-manager.json

```diff
@@ -54,6 +54,14 @@
         },
         "without_repair": {
           "type": "boolean"
+        },
+        "force_tls_disabled": {
+          "type": "boolean",
+          "x-nullable": true
+        },
+        "force_non_ssl_port": {
```

Comment:
`port` is already confusing. Should be rather called "api_port". But let's not change it as it would basically break the contract scylla manager made by introducing this field.

Reply:
That's true, but since we can't change that, I think that we could consider renaming or describing the new flag.

Reply:
OK, maybe it's better to name it "force-non-ssl-session-port" ?

Reply:
changed to "force_non_ssl_session_port"